controllers/AuthController.js

```diff
@@ -73,22 +75,39 @@ export default class AuthController {
     };
 
     const registeredUser = await User.create(user);
+    const {
+      id,
+      firstName: rFirstName,
+      lastName: rLastName,
+      isVerified,
+      type,
+    } = registeredUser;
     const token = Helper.createToken({
-      id: registeredUser.id,
-      firstName: registeredUser.firstName,
-      lastName: registeredUser.lastName,
-      isVerified: registeredUser.isVerified,
+      id,
+      firstName: rFirstName,
+      lastName: rLastName,
+      isVerified,
       email,
-      type: registeredUser.type,
+      type,
     });
     // This line sends the registered user an email
     /* istanbul ignore next-line */
     if (process.env.NODE_ENV !== 'test') {
       EmailVerification.sendVerificationEmail(
-        req, registeredUser.email, registeredUser.firstName,
+        req, email, rFirstName,
         verificationToken
       );
     }
+    let avatar = gravatar.url(email, {
+      s: '200',
```

Comment:
Can the property of this gravatar be more descriptive?

Reply:
what property bro? if you mean the object after email. That is standard I think.